cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -429,67 +471,78 @@ Secret brokersInternalSecret() {
         }
 
         return CompositeFuture.join(result);
-    };
+    }
 
-    private final Future<Void> createOrUpdateTopicOperator(Reconciliation reconciliation, KafkaAssembly kafkaAssembly) {
-        String namespace = kafkaAssembly.getMetadata().getNamespace();
-        String name = kafkaAssembly.getMetadata().getName();
-        log.debug("{}: create/update topic operator {}", reconciliation, name);
-        TopicOperator topicOperator;
-        try {
-            topicOperator = TopicOperator.fromCrd(kafkaAssembly);
-        } catch (Exception e) {
-            return Future.failedFuture(e);
-        }
+    private final Future<TopicOperatorDescription> getTopicOperatorDescription(KafkaAssembly assemblyCm, List<Secret> assemblySecrets) {
```

Comment:
The old `assemblyCm` name seems to be still refering to `ConfigMap` which we no longer use. Renaming it might be better, but that is just a nit pick.

Reply:
Completely agree with you but I left this one just because if you search for `assemblyCm` there are a lot of places where we are still using it. It's not a reason for changing it but I thought to do that in a trivial PR just for renaming in all places in one time.

Reply:
Ok, fine with me.